### ISSUE TITLE
Failed to create MLModel from graph data - with VideoKit 0.0.17 - crash or no image from camera.

### ISSUE THREAD

**Issue by LoopIssuer** [natmlx/videokit#97]
Hi
With the new VideoKit we use HumanTexture. There is an issue - the application crashes after the start or sometimes after the camera start or sometimes there is no crash, but the camera cannot be started after one successful start.
We get 2 kinds of exceptions. Also please take a look into the attached logs:

```
IOException: Sharing violation on path /var/mobile/Containers/Data/Application/C84AD848-7FCB-4493-8F64-1ABA9BABE661/Documents/natml/ecb071a5a8b17481b15a91b5de309920.nml
NatML.MLEdgeModel.CacheSession (NatML.API.Types.PredictorSession session, System.Byte[] graph) (at <00000000000000000000000000000000>:0)
```
```
ArgumentException: Failed to create MLModel from graph data
NatML.MLEdgeModel.Create (System.String tagOrPath, NatML.MLEdgeModel+Configuration configuration, NatML.API.NatMLClient client) (at <00000000000000000000000000000000>:0)
```
[videokitCameraIssue2.txt](https://github.com/natmlx/videokit/files/14042355/videokitCameraIssue2.txt)

[videokitCameraIssue.txt](https://github.com/natmlx/videokit/files/14042311/videokitCameraIssue.txt)

I would like to add, that we already cleared the cache with NatML and Function menu toolbar options - it didn't help.
Tested on: iPhone13,4
OS version: iOS 17.2.1

**Comment by olokobayusuf** [comment]
Will get back to you on this, thanks for reporting it.

**Comment by olokobayusuf** [comment]
Fixed in VideoKit 0.0.18. Feel free to confirm with the latest [VideoKit alpha](https://www.npmjs.com/package/ai.natml.videokit).
